apps/cowswap-frontend/src/legacy/theme/baseTheme.tsx

```diff
@@ -140,7 +140,9 @@ export function themeVariables(darkMode: boolean, colorsTheme: Colors) {
     colorScrollbar: css`
       // Firefox only
       scrollbar-color: ${transparentize(0.5, colorsTheme.text2)} ${colorsTheme.grey1};
-      scroll-behavior: smooth;
+      // TODO: this property breaks Cypress tests for some reason, it prevents scrolling to top
+      // https://github.com/cypress-io/cypress/issues/3200
+      //scroll-behavior: smooth;
```

Comment:
Yes, i faced this one, i would think we solved it long time ago, so long it might be in another project

can we disable it in cypress only? 

https://github.com/cypress-io/cypress/issues/3200#issuecomment-505965470

Reply:
Moved the fix to the test body